### PR TITLE
Add landing pages to 202212 0

### PR DESCRIPTION
**Pull request by michelleegan11**
## PR Description

add landing pages to 202212.0 version of docs

## Checklist
- [ x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/glue-api-guides/202212.0/managing-b2b-account/managing-b2b-account.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Managing B2B account
+description: Describes the endpoints used to manage a B2B account
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-b2b-account/managing-b2b-account.md:3]:
```suggestion
description: This document describes the endpoints used to manage a B2B account
```

File: docs/scos/dev/glue-api-guides/202212.0/managing-b2b-account/managing-b2b-account.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+template: concept-topic-template
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-b2b-account/managing-b2b-account.md:4]: add the `last_updated` tag

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
+template: concept-topic-template
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:4]: add the `last_updated` tag

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
+* Abstract products:
+ 
+  * [Retrieving abstract products](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-abstract-products.html)
+
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:11]:
```suggestion
```

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
+template: concept-topic-template
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
+* Abstract products:
+
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:9]:
```suggestion
```

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
+* Abstract products:
+ 
+  * [Retrieving abstract products](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-abstract-products.html)
+  
+  * [Retrieving image sets of abstract products](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-image-sets-of-abstract-products.html)
+  
+* Concrete products:
+
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:15]:
```suggestion
```

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  * [Retrieving concrete products](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/concrete-products/retrieving-concrete-products.html)
+  
+  * [Retrieving image sets of concrete products](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/concrete-products/retrieving-image-sets-of-concrete-products.html)
+
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:19]:
```suggestion
```

File: docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
```
Comment (VadymSachenko) [inline docs/scos/dev/glue-api-guides/202212.0/managing-products/managing-products.md:3]:
```suggestion
description: This document describes the endpoints used to manage products
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

**Comment by VadymSachenko** [comment]
@michelleegan11 please add your pages to the scos sidebar if you haven't done it yet.

**Comment by michelleegan11** [comment]
> @michelleegan11 please add your pages to the scos sidebar if you haven't done it yet.

I've added them in this PR, thanks
https://github.com/spryker/spryker-docs/pull/1494/files

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**